src/main/java/org/sefglobal/scholarx/service/EmailService.java

```diff
@@ -1,37 +1,32 @@
 package org.sefglobal.scholarx.service;
-import org.sefglobal.scholarx.model.Email;
-import org.sefglobal.scholarx.util.EmailUtil;
+
+import org.sefglobal.scholarx.model.Mail;
 import org.springframework.stereotype.Service;
 
-import javax.mail.MessagingException;
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
 @Service
 public class EmailService {
 
-    private final EmailUtil emailUtil;
-
-    public EmailService(EmailUtil emailUtil) {
-        this.emailUtil = emailUtil;
-    }
+    private SimpleJavaMail simpleJavaMail;
 
-    public Email sendEmail(String emailAddress, String subject, String message, boolean showButton) throws IOException, MessagingException {
-        Email email = new Email();
+    public void sendEmail(String name, String emailAddress, String subject, String message, boolean showButton) throws IOException {
+        Mail email = new Mail();
+        email.setName(name);
         email.setEmail(emailAddress);
         email.setSubject(subject);
         email.setMessage(message);
 
         Map<String, Object> model = new HashMap<String, Object>();
+        model.put("name", name);
```

Comment:
We don't need a name, right?

Reply:
Here also the same reason as https://github.com/sef-global/scholarx/pull/345#discussion_r925098013
But I think the `name` variable should be renamed to `recipientName` or something. Is it?

Reply:
I added a comment. Even if the name is needed you don't need to put it into this hashmap. It is there to build the email template.

Reply:
Fixed with [be9c7f5](https://github.com/sef-global/scholarx/pull/345/commits/be9c7f59367a13dff8d8dc95bfb24ba32c00bc4e)